src/WorkItemMigrator/WorkItemImport/WitClient/WitClientUtils.cs

```diff
@@ -25,9 +26,9 @@ public WitClientUtils(IWitClientWrapper witClientWrapper)
 
         public delegate V IsAttachmentMigratedDelegate<in T, U, out V>(T input, out U output);
 
-        public WorkItem CreateWorkItem(string type)
+        public WorkItem CreateWorkItem(string type, DateTime createdDate = default, string createdBy = "")
```

Comment:
use null instead of default

Reply:
Fixed by https://github.com/solidify/jira-azuredevops-migrator/pull/648/commits/4f6ee65a7c91eaa61a952ca35f3b40df31fcb569